cheroot/test/test_conn.py

```diff
@@ -8,8 +8,12 @@
 import six
 import pytest
 
-from cheroot._compat import HTTPConnection, HTTPSConnection, NotConnected, BadStatusLine
-from cheroot._compat import ntob, urlopen
+from cheroot._compat import HTTPConnection
```

Comment:
From the link you provided:
> It's okay to say this though:
> 
> from subprocess import Popen, PIPE

so for `from ... import ...` it's OKay to list a few of them

they may be split into several lines using several `from ... import ...` statements or just multilined like
```python
from some_lib import (
    importable1, importable2,
    importable3, importable4,
    etc,
)
```